### ISSUE TITLE
PSM New status type - Expired (LP) (On PROD)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#12137]
### Create new valid status on PSM for EXPIRED
- ASAP

**Comment by TracyLinder** [comment]
@Nadin-Nader we need to pick this up ASAP
FYI @Lpriya-LIT and @SubhaShroff

**Comment by SubhaShroff** [comment]
@TracyLinder  I will look at it, today.

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Committed changes, Ready for Dev and Test Deployment
SCTASK0309311

**Comment by Nadin-Nader** [comment]
Sent message to Kyle that this is ready for UAT on TEST

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Created Prod prootion request - SCTASK0313008

**Comment by Nadin-Nader** [comment]
Still not deployed to PROD

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Changes deployed to Prod , Ready for UAT

**Comment by Nadin-Nader** [comment]
Sent Message to Kyle and ROW Team to please UAT and let us know if this is good to close or not now that it is on PROD. Waiting on Response

**Comment by Nadin-Nader** [comment]
Sent Message to Kyle and ROW Team to please UAT and let us know if this is good to close or not now that it is on PROD. Waiting on Response

**Comment by Nadin-Nader** [comment]
Per Kyle this is good to go, Closing.

> [8:19 AM] Kampe, Kyle
>     I UAT'd 12137 as part of the last batch of UATs I did, and it's good to go. Thanks!
